src/ucp/wireup/select.c

```diff
@@ -606,10 +606,13 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
                 continue;
             }
 
-            score        = criteria->calc_score(wiface, md_attr, address, ae,
-                                                criteria->arg);
-            priority     = iface_attr->priority + ae->iface_attr.priority;
-            is_reachable = 1;
+            is_prioritized = select_params->ep_init_flags &
+                             UCP_EP_INIT_FLAG_PRIORITIZED;
```

Comment:
minor: the variable looks redundant to me or it's initialization should be moved before top the level loop